.github/workflows/markdown-link-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Check Markdown links
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+  
+jobs:
+  markdown-link-check:
+    name: Check for broken links
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: Run link check 
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-verbose-mode: 'yes'
+        folder-path: 'src/pages/en/'
```

Comment:
Seems that this is going to fail due to now new line @criadoperez

Reply:
Could you please elaborate? Sorry, but I'm don't understand your point here.
I testing it running [here[(https://github.com/criadoperez/stacks-network_docs/runs/5831596238?check_suite_focus=true) where it run ok and detected all the failed links that have already been fixed since that run.

Reply:
The last issues detected by the markdown link check have been fixed on this other [PR](https://github.com/stacks-network/docs/pull/1343)